web-frontend/src/i18n/i18n.ts

```diff
@@ -23,7 +23,11 @@ class I18n {
   public t(key: string): string {
     let copy = this.copy;
     // we are using pick in order to make life easier when the day for plurals and copy with params arrives
-    return dot.pick(key, copy);
+    const result = dot.pick(key, copy);
+    if (result === undefined) {
```

Comment:
Are we sure about that!? How a result could be **undefined**!? 🤔

Reply:
if it's not found. I'd like to show the key if the translation doesn't exists:
- to avoid blank
- to ease debugging

Reply:
Ah! I see what you mean. All good!
However, if you could do in a simple way:
```tsx

const result = dot.pick(key, copy);
return result ?? key
```

Reply:
yes, I'll try this in another PR